src/oabot/templates/change.html

```diff
@@ -1,4 +1,13 @@
 <!doctype html>
+{% macro circle_color(policy) -%}
+    {% if policy == 'can' %}
+        <div class="circle-green"></div> Allowed
+    {% elif policy == 'restricted' %}
+        <div class="circle-orange"></div> Restricted
+    {% elif policy == 'cannot' %}
+        <div class="circle-red"></div> Forbidden
+    {% endif %}
+{%- endmacro %}
```

Comment:
actually sometimes this status can be 'unknown' too, I forgot to tell you :-/

Reply:
Ah! I'll add an 'else' for 'unknown' and anything else that might happen to come through.